pysrc/bytewax/run.py

```diff
@@ -0,0 +1,237 @@
+import argparse
+import importlib
+import pathlib
+import os
+import sys
+
+from bytewax.recovery import SqliteRecoveryConfig
+
+from .bytewax import cluster_main, spawn_cluster
+
+__all__ = ["cluster_main"]
+
+
+class ImportFromStringError(Exception):
```

Comment:
The idea is that this is a user facing error, and it's usually better to see:

```
CustomError: Error while parsing the import string, is it in the right format?
```

rather than a criptic
```
TypeError: NoneType is not a function
```

or something like that. So it is not tagged for catching, but used to wrap possible exceptions into a more comprehensible error for the user. Although the name of the exception is a bit cryptic, I'm going to change it to "BytewaxRunError"

Reply:
This is solely taste: I do think it's worthwhile to catch and re-raise errors with more descriptive error messages. But I don't think that this needs a custom exception class unless you specifically want to facilitate catching of this class of error programmatically or none of the exception types really describe what's going on. `ValueError` with a good message feels fine to me here. You're welcome to keep it, though.

Reply:
Makes sense, since we don't need users to catch all the possible errors here, base python exceptions with custom messages will do the job just fine. Fixed